src/pe/import.rs

```diff
@@ -26,41 +60,37 @@ impl<'a> HintNameTableEntry<'a> {
 #[derive(Debug, Clone)]
 pub enum SyntheticImportLookupTableEntry<'a> {
     OrdinalNumber(u16),
-    HintNameTableRVA ((u32, HintNameTableEntry<'a>)), // [u8; 31] bitfield :/
+    HintNameTableRVA ((usize, HintNameTableEntry<'a>)), // [u8; 31] bitfield :/
 }
 
 #[derive(Debug)]
 pub struct ImportLookupTableEntry<'a> {
-    pub bitfield: u32,
```

Comment:
If we're going to  delete `bitfield` (which I think is fine to do), then there doesn't seem much point having this struct at all.